cypress/integration/buy-shirt.spec.ts

```diff
@@ -3,7 +3,7 @@ import {
   ProductListPage, ShopingCartPage,
 } from "../page/index";
 
-const menuContentPage = new MenuContentPage("http://automationpractice.com/");
+const menuContentPage = new MenuContentPage();
```

Comment:
Lets try to keep the AAA pattern as it is on the dresses-list.spec.ts

1. The page object should be declared as let
2. On the arrange section, the pages should be initialized using beforeAll() 

😄